token/core/fabtoken/loaders.go

```diff
@@ -53,6 +62,7 @@ func (s *VaultPublicParamsLoader) Load() (*PublicParams, error) {
 	return pp, nil
 }
 
+// ForceFetch returns the PublicParams associated with fabtoken
```

Comment:
maybe just add a description of how is this different from simple Load

Reply:
The load specifies that it looks first in the cache, but I will extend the description here.